node/node.go

```diff
@@ -224,6 +225,8 @@ type NodeController struct { // nolint:golint
 	nodeStatusUpdateErrorHandler ErrorHandler
 
 	chReady chan struct{}
+	chDone  chan struct{}
```

Comment:
Can you add a comment about what this does?

Reply:
Done.